modules/file_functions.py

```diff
@@ -0,0 +1,44 @@
+"""
+Support for os and other modules which written for avoiding repetitive code.
+"""
+import os
+from modules import constants, run_command
+
+
+def check_file_existence(file_path, debug, container_name):
+    """This function checks if the file is exist in the system."""
+    exist = False
+    if container_name:
+        cat_file_command = f'cat {file_path}'
+        pipe_cat_file = run_command.command_output(cat_file_command, debug, container_name)
+        if pipe_cat_file.stdout:
+            exist = True
+    else:
+        if os.path.isfile(file_path):
+            exist = True
+    return exist
+
+
+def file_content(file_path, debug, container_name):
```

Comment:
```suggestion
def get_file_content(file_path, debug, container_name):
```
Make sure to change in all relevant references to the function